scss/_alert.scss

```diff
@@ -54,18 +54,13 @@
 
 
 // scss-docs-start alert-modifiers
-// Generate contextual modifier classes for colorizing the alert.
-
-@each $state, $value in $theme-colors {
```

Comment:
Haven't looked at it yet but I'm afraid that https://deploy-preview-35857--twbs-bootstrap.netlify.app/docs/5.2/customize/sass/#add-to-map won't work anymore.